include/mrdocs/Metadata/Info.hpp

```diff
@@ -200,7 +132,7 @@ struct MRDOCS_VISIBLE
     class is a certain type.
 */
 template <InfoKind K>
-struct IsInfo : Info
+struct TypedInfo : Info
```

Comment:
I prefer we don't change the name of this, or use `Type` anywhere in `InfoNodes.inc`. We already have `TypeInfo`, so this makes it pretty confusing. Prefer `Kind` for the names of the macro parameter.

Reply:
> We already have TypeInfo, so this makes it pretty confusing. 

Mmm... Good thinking. I forgot about TypeInfo. That *is* confusing. You can also see why `IsInfo` is confusing: it's not a trait, and it has nothing to do with what it does. Is there another name you could suggest? 

 -InfoType
- InfoCategory
- ClassifiedInfo
- SpecificInfo
- CategorizedInfo
- TypeSpecificInfo
- TypedMetadata
- InfoDiscriminator
- InfoVariant
- TypeDefinedInfo
- ...

> Prefer Kind for the names of the macro parameter

I don't know what you mean here.

Reply:
@alandefreitas `IsInfoKind`? Regarding the name of the macro parameter, I'd use `KIND` rather than `TYPE`.

Reply:
I think the `Is*` prefix makes it look like a trait, though. Maybe that's a convention somewhere else, but I've never seen an `is*` class that's not a trait.

Reply:
`InfoBase`? `InfoCommon`? `InfoCommonBase`? Anything that doesn't contain "type" :)

Reply:
I think I like InfoCommonBase for now.

Reply:
Done